examples/ElmLogo.elm

```diff
@@ -99,25 +92,33 @@ loadModel withTangents url =
 -- UPDATE
 
 
+type alias Size =
+    { width : Int, height : Int }
+
+
 type Msg
-    = Tick Float
+    = Tick Posix
     | LoadObj String (Result String (Dict String (Dict String Mesh)))
     | Zoom Float
-    | MouseMove Mouse.Position
-    | MouseDown Mouse.Position
+    | MouseMove Int Int
+    | MouseDown Int Int
     | MouseUp
-    | DiffTextureLoaded (Result String GL.Texture)
-    | NormTextureLoaded (Result String GL.Texture)
-    | ResizeWindow Window.Size
+    | DiffTextureLoaded (Result String Texture)
+    | NormTextureLoaded (Result String Texture)
+    | ResizeWindow Int Int
     | SelectMesh String
     | SetUseTangent Bool
 
 
+type alias CameraInfo =
+    { projection : Mat4, view : Mat4, viewProjection : Mat4, position : Vec3 }
+
+
 update : Msg -> Model -> ( Model, Cmd Msg )
 update msg model =
     case msg of
         Tick dt ->
-            ( { model | time = model.time + dt / 1000 }, Cmd.none )
+            ( { model | time = (posixToMillis dt |> toFloat) / 1000 }, Cmd.none )
```

Comment:
Once you switch to `onAnimationFrameDelta`, this no longer needs to use `posix` and the `model.time + ..` can be added back. In this example we only care about the total time, but in most animations and games we care about the delta time, so I'd prefer if this example also follows that

Reply:
Ahh, right that explains it... thought that had been deprecated but must have missed it